### DISCUSSION TITLE
how to invite bot?

### DISCUSSION THREAD

**Discussion by Nirav-Datta** [kozabrada123/Aternos-On-Discord-Update#5] [Q&A]
How to invite bot to discord server?

**Comment by kozabrada123** [comment]
1. Go to https://discord.com/developers/applications/
2. Click on your bot
3. Go to OAuth2 on the left
4. Go to the URL Generator
5. Under Scopes, select bot
6. Scroll down to bot permissions
7. Tick at least Send Messages and Read Message History
8. Copy the generated url at the bottom
9. Go to the url in your browser
10. Select a server you have permissions for
11. Click Authorize
